.claude/skills/check-pr/scripts/fetch-jira-tree.sh

```
@@ -0,0 +1,133 @@
+  echo '{"error": "JIRA_TOKEN environment variable is not set"}' >&2
+  exit 1
+fi
+
+fetch_issue() {
+  local key="$1"
+  local fields="summary,issuetype,priority,status,parent,fixVersions,customfield_12311140,assignee,description"
+
+  curl -sS --fail-with-body \
+    -H "Authorization: Bearer $JIRA_TOKEN" \
+    -H "Content-Type: application/json" \
+    "$api_url/issue/$key?fields=$fields" 2>/dev/null || echo '{"error": "failed to fetch '$key'"}'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)